src/planning/launch/eufs.launch.py

```diff
@@ -93,7 +93,7 @@ def generate_launch_description():
             DeclareLaunchArgument(
                 "pre_defined_velocity_planning",
                 description="A pre-defined velocity planning value",
-                default_value="5",
+                default_value="1",
```

Comment:
Why did you change from 1 to 2? Pretty sure it was working with 2

Reply:
Also, ran the simulation, I think it is still using 2 anyway for some reason

Reply:
Isso foi só para testar quando a simulação não funcionada e não reverti. Vou resolver